views/js/qtiCreator/helper/itemLoader.js

```diff
@@ -33,6 +33,12 @@ define([
         return uri.substring(pos + 1, pos + 1 + qtiIdentifier.maxQtiIdLength);
     };
 
+    const _generateUniqueNumericIdentifier = function _generateUniqueNumericIdentifier() {
```

Comment:
Lets move this function to some helper file which can be imported, we even can't unit test it now.

Reply:
And I think its good idea to add unit test, this is core part of this change.